src/components/App.tsx

```diff
@@ -0,0 +1,37 @@
+
+
+import { Route, Routes } from 'react-router-dom';
+import Family from './Family/Family';
+import Sidebar from './Sidebar/Sidebar';
+import Home from '../pages/Home';
+import Friends from './Friends/Friends';
+import CloseFriends from './CloseFriends/CloseFriends';
+import JustPeople from './JustPeople/JustPeople';
+import Footer from './Footer/Footer';
+import Header from './Header.tsx/Header';
+import s from './App.module.css';
+
+function App() {
+  return (
+    <>
+      <Header />
+      <div className={s.wrap} >
+      <Sidebar/>
+      
+      <main>
```

Comment:
where is main component?
make routes content of the main, not children

Reply:
Done